src/main/java/com/slcube/shop/business/category/domain/Category.java

```diff
@@ -21,8 +25,16 @@ public class Category extends BaseEntity {
     @Column(nullable = false)
     private String categoryName;
 
+    @OneToMany(mappedBy = "category")
+    private List<Item> items = new ArrayList<>();
+
     @Builder
     private Category(String categoryName) {
         this.categoryName = categoryName;
     }
+
+
+    public void updateCategory(CategoryUpdateRequestDto requestDto) {
+        this.categoryName = requestDto.getCategoryName();
+    }
```

Comment:
category 이름을 변경하는 dto class가 여러개가 되면 dto class 별로 updateCategory method가 중복이 됩니다.
```suggestion
    public void updateCategory(String newCategoryName) {
        this.categoryName = newCategoryName;
    }
```

위와 같이 변경하는건 얼떨까요??